GuessTheWord-Starter/app/src/main/java/com/example/android/guesstheword/screens/score/ScoreViewModelFactory.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.android.guesstheword.screens.score
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import java.lang.IllegalArgumentException
+
+class ScoreViewModelFactory(private val finalScore: Int) : ViewModelProvider.Factory {
+
```

Comment:
Do you prefer to have a new line at the start and end of classes? If not, please remove those.
Also, try to format indentation of this file. Please fix that in a last commit